README.md

```diff
@@ -143,34 +123,43 @@ This workflow also allows you to modify or customize the HarbourBridge codebase.
 To use the tool on a PostgreSQL database called mydb, run
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump
+pg_dump mydb > mydb.pg_dump
+harbourbridge schema -source=postgresql < mydb.pg_dump
```

Comment:
This command is for schema mode, but the text below (the paragraph starting "This command ...") describes what happens for schema-and-data mode.

We should be consistent here. A couple of options:

Option 1:  Just focus on schema migration. Suggest changing the text above the command to something like:
```
 To use the tool for schema-migration on a PostgreSQL database called mydb, run
```
and then in the description below lay out what the tool does.

Option 2: Focus on schema and data migration. In this case change the command and keep the existing descriptions.

I've a slight preference for Option 2 because it shows the full range of HarbourBridge functionality, and is a good starting point for first users. But I don't care strongly.

Reply:
I agree that for someone just starting with HB, it's better to describe the full functionality since we describe the various subcommands separately as well. Have updated to use `schema-and-data`.